src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  width: $width;
+  gap: $gap;
+  padding: $padding;
+
+  &__day {
+    width: $size;
+    height: $size;
+    background: #eee;
+    border: 1px solid black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+
+    @for $index from 1 through 31 {
+      &:nth-child(#{$index})::before {
+          content: "#{$index}";
+        }
+
+        &:hover {
+          cursor: pointer;
```

Comment:
you can add cursor: pointer to element styles (not on hover)

Reply:
fixed